definitions.yaml

```diff
@@ -256,7 +256,7 @@ definitions:
         description: Expiration time (from creation) in hours
       cn_prefix:
         type: string
-        description: The common name prefix of the certificate subject.
+        description: The common name prefix of the certificate subject. This only allows characters that are usable in domain names (`a-z`, `0-9`, and `.-`, where `.-` must not occur at the start nor end).
       certificate_organizations:
```

Comment:
Nitpick 
I think it is `or` instead of `nor` at the end of the phrase
`at the start or end`

Reply:
🇬🇧 driveby:

```
where `.-` must not occur at either the start or the end).
```

would be how I would right it. `nor` is super old school.